Resources/Prototypes/_DV/Guidebook/CompanyPolicy.yml

```
@@ -0,0 +1,72 @@
+# Main Company Policy Directory
+- type: guideEntry
+  id: companyPolicies
+  name: Company Policy
+  text: "/ServerInfo/Guidebook/_DV/SPLAW/empty.xml"
```

doesn't need quotes
same for rest of file
```suggestion
  text: /ServerInfo/Guidebook/_DV/SPLAW/empty.xml
```
Also shouldn't this entry itself contain the preface instead of the preface being its own entry and this one being blank?